daft/dataframe/dataframe.py

```diff
@@ -422,9 +431,7 @@ def read_parquet(cls, path: str) -> DataFrame:
             schema=schema,
             predicate=None,
             columns=None,
-            source_info=ParquetSourceInfo(
-                filepaths=filepaths,
-            ),
+            source_info=ParquetSourceInfo(filepaths=filepaths, sizes_bytes=sizes_bytes),
```

Comment:
One potential  issue I see here is that, this is the sizes for the parquet files with compressed columns and this maybe could blow up when decompressed. A good way to test this would be to make a parquet file with a single column and a huge number of rows of the exact same value. The file size will be much less than the decompressed size.

Reply:
I guess this is also a problem with files with extensions like gz.